pkg/kubelet/dockertools/manager_test.go

```diff
@@ -712,6 +713,15 @@ func TestSyncPodDeletesWithNoPodInfraContainer(t *testing.T) {
 		},
 	}
 
+	// (random-liu) Seems that need  container map here
```

Comment:
That's really wired. It seems that in my old implementation, it hit here. But now, I remove this and test again, it's OK now. Maybe I can remove this.